commit c92ba33505ad98ed42ebb7fab486d8ef201a1ed3
Author: Venky Shankar <vshankar@redhat.com>
Date:   Thu Apr 30 05:57:40 2020 -0400

    mgr: set `task_dirty_status` on reconnect
    
    ... so that upon reconnect the manager client updates manager
    with whatever is available in ->task_status field.
    
    Signed-off-by: Venky Shankar <vshankar@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index c63d54f2fee..4ac88dd99d4 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -184,6 +184,7 @@ void MgrClient::reconnect()
   if (service_daemon) {
     daemon_dirty_status = true;
   }
+  task_dirty_status = true;
 
   // Don't send an open if we're just a client (i.e. doing
   // command-sending, not stats etc)
